app/routes/planet_routes.py

```diff
@@ -0,0 +1,93 @@
+from flask import Blueprint, abort, make_response, request, Response
+from ..models.planet import Planet
+from ..db import db
+from sqlalchemy import asc, desc
+
+planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
+
+@planets_bp.post("")
+def create_planet():
+    request_body = request.get_json()
+    name = request_body["name"]
+    surface_area = request_body["surface_area"]
+    moons = request_body["moons"]
+    distance_from_sun = request_body["distance_from_sun"]
+    namesake = request_body["namesake"]
```

Comment:
What kind of error could be raised by this code? What's a way we could handle it?